noxfile.py

```diff
@@ -52,11 +52,14 @@ def test(session):
 
     session.install("-rdev-requirements.txt", ".")
     session.run("pip", "freeze")
+    memray_supported = not session.run("python", "-c", '"import pytest_memray"')
+
     session.run(
         "pytest",
         "-v",
         "-s",
         "-rs",
+        *("--memray", "--hide-memray-summary") if memray_supported else (),
```

Comment:
I got stuck marveling that the order of precedence between splat and ternary if lets this work. Probably clearer to put it in a variable above and then use it with splat here